documentation/modules/con-creating-managing-connectors.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-connect.adoc
+
+[id='con-creating-managing-connectors-{context}']
+
+= Creating and managing connectors
+
+Once you have created a container image for your connector plug-in, you need to create a connector instance in your Kafka Connect cluster. You can then configure, monitor, and manage a running connector instance. For example, you can:
+
+* Check the status of a connector instance
+* Increase or decrease the number of tasks
+* Restart failed tasks
+* Pause a connector instance
+* Delete a connector instance
+
+{ProductName} provides two different APIs for creating and managing connectors:
+
+* `KafkaConnector` resources (referred to as `KafkaConnectors`)
+* Kafka Connect REST API
+
+For security reasons, we recommend that you use `KafkaConnectors` rather than the Kafka Connect REST API. `KafkaConnectors` use Kubernetes RBAC controls to secure the Kafka Connect REST API and lock down its resources.
+
+NOTE: You need to use the Kafka Connect REST API for restarting failed tasks and pausing and resuming connectors. Future {ProductName} releases might add this functionality to `KafkaConnectors`.
+
+.`KafkaConnector` resources
+
+`KafkaConnectors` allow you to create and manage source and sink connector instances for Kafka Connect in a Kubernetes-native way, so an HTTP client such as cURL is not required. Like other Kafka resources, you declare a connector’s desired state in a `KafkaConnector` YAML file that is deployed to your Kubernetes cluster to create a corresponding connector instance. 
```

Comment:
This is not incorrect, but (as a general point) I don't especially like the terminology of "deploying a YAML file to Kubernetes". It's only a file if you're using a file to represent the resource locally. There are other ways of interacting with the Kubernetes API server which aren't so much about files (e.g. `kubectl edit`, various consoles etc).